_templates/playground/new/vue.md.ejs.t

```diff
@@ -0,0 +1,27 @@
+---
+arbitrary: <% pascalName = h.changeCase.pascal(name); componentName = h.changeCase.pascal(path.split('/').pop()) %>
+to: "<%= `static/usage/v${version}/${name.replace('ion-', '')}/${path}/vue.md` %>"
+sh: "echo `\nGeneration complete! To use this new component in a docs markdown file, include\nthe following:\n\n## <%= componentName %>\n\nimport <%= componentName %> from '@site/static/usage/v7/<%=name.replace('ion-', '') %>/<%= path %>/index.md';\n\n<<%= componentName %> />\n`"
```

Comment:
Not sure if this intentional, but the terminal output says "shell: echo"

```
✔ Which component is this playground for? · ion-datetime
✔ What should the playground path be? · liam
✔ Select the Ionic Framework version for the playground · 7
✔ Do you want CSS scaffolded? · No / Yes

Loaded templates: _templates
       added: static/usage/v7/datetime/liam/angular/example_component_css.md
       added: static/usage/v7/datetime/liam/angular/example_component_html.md
       added: static/usage/v7/datetime/liam/demo.html
       added: static/usage/v7/datetime/liam/index.md
       added: static/usage/v7/datetime/liam/javascript.md
       added: static/usage/v7/datetime/liam/react/main_css.md
       added: static/usage/v7/datetime/liam/react/main_tsx.md
       added: static/usage/v7/datetime/liam/vue.md
       shell: echo `
Generation complete! To use this new component in a docs markdown file, include
the following:

## Liam

import Liam from '@site/static/usage/v7/datetime/liam/index.md';

<Liam />
`
```

Reply:
Yeah, this was the only way I could find to do this output without recreating the whole prompts file. (Basically, we're using a simple setup for the prompts and will need a more complex way of doing everything to do this in a less hacky way.) Like I said in response to Amanda's comment, this is intended as an MVP of this particular feature.